webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -206,23 +241,24 @@ export class GenericModal extends React.PureComponent<Props, State> {
                     className='GenericModal__wrapper-enter-key-press-catcher'
                 >
                     <Modal.Header closeButton={true}>
-                        <div className='GenericModal__header__text_container'>
+                        <div
+                            className='GenericModal__header__text_container'
+                        >
                             {this.props.compassDesign && (
                                 <>
                                     {headerText}
                                     {this.props.headerInput}
                                 </>
                             )}
-
                             {
                                 this.props.modalSubheaderText &&
                                 <div className='modal-subheading-container'>
-                                    <p
+                                    <div
```

Comment:
replaced `p` tag with `div` since we were breaking the rule that says no `div` should be children of `p`-